Dockerfile

```diff
@@ -20,16 +22,37 @@ WORKDIR /
 
 RUN mkdir project
 
-RUN git clone https://github.com/dpirkl/lecture-sts.git
+RUN git clone https://github.com/OrbitPeppermint/lecture-sts.git
 
 RUN mv lecture-sts/* project/
 
 RUN rm -r lecture-sts
 
+# added  for seamlessm4t installation
+RUN git clone https://github.com/facebookresearch/seamless_communication.git
+
+
 WORKDIR /project
 
 # Install requirements to perform the translation
 RUN pip install -r requirements.txt
 
+# added for seamlessm4t
+#UN pip install -r dev_requirements.txt
+
+# Install latest Whisper Version
+RUN pip install -U openai-whisper
```

Comment:
If you can install whisper via pip, maybe we should replace 'git+https://github.com/openai/whisper.git' with it in the requirements.txt.